tests/rados/test_bluestoretool_workflows.py

```diff
@@ -463,28 +463,27 @@ def run(ceph_cluster, **kw):
             log.info(out)
             assert "success" in out
 
-            if rhbuild.split(".")[0] != "8":
-                # Execute ceph-bluestore-tool qfsck --path <osd_path>
-                osd_id = random.choice(osd_list)
-                log.info(
-                    f"\n --------------------"
-                    f"\n Running quick consistency check for OSD {osd_id}"
-                    f"\n --------------------"
-                )
-                out = bluestore_obj.run_quick_consistency_check(osd_id=osd_id)
-                log.info(out)
-                assert "success" in out
+            # Execute ceph-bluestore-tool qfsck --path <osd_path>
```

Comment:
Is this now available with all the releases?

Reply:
This was a regression bug in only 8.0